python/lsst/ts/standardscripts/auxtel/latiss_take_sequence.py

```diff
@@ -0,0 +1,295 @@
+        super().__init__(
+            index=index,
+            descr="Perform a sequence of images with LATISS.",
+        )
+
+        latiss_usage = None if add_remotes else LATISSUsages.DryTest
+
+        atcs_usage = None if add_remotes else ATCSUsages.DryTest
+
+        # Instantiate latiss. We need to do this after the call to
+        # super().__init__() above. We can also pass in the script domain and
+        # logger to both classes so log messages generated internally are
+        # published to the efd.
+        self.latiss = LATISS(
+            domain=self.domain, intended_usage=latiss_usage, log=self.log
+        )
+        self.atcs = ATCS(domain=self.domain, intended_usage=atcs_usage, log=self.log)
+
+        self.run_started = False
+
+    @classmethod
+    def get_schema(cls):
+        schema_yaml = """
+            $schema: http://json-schema.org/draft-07/schema#
+            $id: https://github.com/lsst-ts/ts_externalscripts/auxtel/latiss_take_sequence.yaml
+            title: LatissAcquireAndTakeSequence v1
+            description: Configuration for LatissAcquireAndTakeSequence Script.
+            type: object
+            properties:
+              filter_sequence:
+                description: Filters for exposure sequence. If a single value is specified then
+                   the same filter is used for each exposure.
+                anyOf:
+                  - type: array
+                    minItems: 1
+                    items:
+                      type: string
+                  - type: string
+                default: empty_1
+
+              grating_sequence:
+                description: Gratings for exposure sequence. If a single value is specified then
+                   the same grating is used for each exposure.
+                anyOf:
+                  - type: array
+                    minItems: 1
+                    items:
+                      type: string
+                  - type: string
+                default: empty_1
+
+              exposure_time_sequence:
+                description: Exposure times for exposure sequence (sec). Each exposure requires
+                   a specified exposure time.
+                anyOf:
+                  - type: array
+                    minItems: 1
+                    items:
+                      type: number
+                      minimum: 0
+                  - type: number
+                    minimum: 0
+                default: 2.
+
+              reason:
+                description: Optional reason for taking the data.
+                anyOf:
+                  - type: string
+                  - type: "null"
+                default: null
+
+              program:
+                description: >-
+                  Optional name of the program this dataset belongs to.
+                anyOf:
+                  - type: string
+                  - type: "null"
+                default: null
+
+              do_track_target:
```

Comment:
I think you should remove this option. This Script should only run if we are tracking.